finetuner/data.py

```diff
@@ -222,6 +222,31 @@ def parse(self):
                 yield Document(chunks=[doc1, doc2], tags={DEFAULT_TAG_SCORE_KEY: col3})
 
 
+class DataSynthesisParser(_CSVParser):
+    """
+    CSV has either one column or one row, each item in the CSV represents a single
+    document so the structure of the CSV file is not important.
```

Comment:
why csv has one row or one column?

Reply:
The structure of data for synthesis jobs is just a list of texts, there is no need for the CSV file to have a complicated structure and formatting it as either a single row or column is the simplest way